### PR TITLE
Palo to fortigate fixes

### PR DESCRIPTION
**Pull request by mymyke**
simmular to my palo_to_checkpoint pull request there were also errors with palo to fortigate converstion. 
I added the same input file as in the other pull request and also added the output files.

### CODE REVIEW

File: resources/src_vendor/palo/palo_config_coverter.py [hunk 1]
```diff
@@ -61,6 +62,9 @@ def service(self):
 
             
             if self.vendor == 'forti':
+                if "," in destination_port:
```
Comment (VahidTa) [inline resources/src_vendor/palo/palo_config_coverter.py:65]: you don't need the `if` condition, use `.replace()`

File: resources/src_vendor/palo/palo_config_coverter.py [hunk 2]
```diff
@@ -61,6 +62,9 @@ def service(self):
...
+                    # on a fortigate a service with multiple ports is sepperated by a space not a semicolon
```
Comment (VahidTa) [inline resources/src_vendor/palo/palo_config_coverter.py:66]: please correct the spelling.

File: resources/src_vendor/palo/palo_config_coverter.py [hunk 3]
```diff
@@ -148,6 +155,14 @@ def address(self):
 
 
             if self.vendor == 'forti':
+                try:
+                    ip_network(address_ip, strict=True)
+                    # address_ip is a valid network address (or a valid host with /32 mask) -->nothing to do
+                    address_ip = address_ip.split("/")[0] +" "+str(ip_network(address_ip).netmask)
```
Comment (VahidTa) [inline resources/src_vendor/palo/palo_config_coverter.py:161]: if the format is "192.168.1.0/24"
do this:

```python

address_ip = ip_network(address_ip, strict=True).with_netmask
```

File: resources/src_vendor/palo/palo_config_coverter.py [hunk 4]
```diff
@@ -148,6 +155,14 @@ def address(self):
...
+                except ValueError:
+                    # address_ip is a palo specific host address that also has a network mask
+                    # --> convert to Host
+                    address_ip = address_ip.split("/")[0]+" 255.255.255.255"
```
Comment (VahidTa) [inline resources/src_vendor/palo/palo_config_coverter.py:165]: please share the palo address format

**Review by VahidTa — CHANGES_REQUESTED** [review]

File: resources/src_vendor/palo/palo_config_coverter.py (hunk 4 above)
Comment (mymyke) [reply resources/src_vendor/palo/palo_config_coverter.py:166]: In Palo alto you have can have 3 kinda different formats.
1. just the IP of an Host. for example "192.168.1.123"
2. The Network adress with a subnet mask in cidr notation. For example "192.168.1.0/24"
3. The IP address of an host with a subnet mask indicating in wich subnet the IP is used. As far as i am aware this is just for readability, and wont get used. for example "192.168.1.123/24"

Case 3 is what gets handled here ( in the except ValueError case).

This was the "cleanest" way i found to convert it into a simple host address.

**Comment by VahidTa** [comment]
put back .keep file under "exported"

**Comment by VahidTa** [comment]
Please remove the ".keep" file under the "exported" directory. Otherwise, I can't merge it and I'll close the MR

**Review by VahidTa — APPROVED** [review]

**Comment by VahidTa** [comment]
@mymyke  rebase locally and push again.

**Comment by VahidTa** [comment]
@mymyke please rebase this PR locally and push again. otherwise, I'll close this.

**Comment by VahidTa** [comment]
I am closing this because of the code conflict.

**Pull request closed without merging**